.gitignore

```diff
@@ -1,2 +1,3 @@
 target/
 .idea
+.vscode/
```

Comment:
nit: add a newline at the end of this file.

Reply:
Will be fixed in #2